bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -3881,6 +3889,12 @@ void ReplicaImp::executeNextCommittedRequests(concordUtils::SpanWrapper &parent_
   if (isCurrentPrimary() && requestsQueueOfPrimary.size() > 0) tryToSendPrePrepareMsg(true);
 }
 
+void ReplicaImp::TryToGotoNextView() {
+  if (complainedReplicas.hasQurumToLeaveView()) {
+    GotoNextView();
+  }
```

Comment:
Add a INFO log statement in the else ("insufficient quorum for moving to next view")